src/ontology/uberon-edit.obo

```diff
@@ -226353,6 +226354,29 @@ is_a: UBERON:7500094 ! tubercle of calcaneus
 property_value: http://purl.org/dc/elements/1.1/date 2021-08-04T00:27:56Z xsd:dateTime
 property_value: term:editor http://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500098
+name: sagittal crest of skull
+def: "A ridge of bone running lengthwise along the midline of the top of the skull of many mammalian and reptilian skulls and serves as an attachment site for the temporalis muscle[WP, modified]." [Wikipedia:Sagittal_crest]
+synonym: "sagittal ridge" RELATED []
+xref: Wikipedia:Sagittal_crest
+is_a: UBERON:0005913 ! zone of bone organ
```

Comment:
I think you mean to have UBERON:7500099 here

This seems like a good candidate for an equivalence axiom

Reply:
This def is for the skull specifically, even though the wiki article just says "sagittal_crest".

Reply:
@cmungall I think I fixed it.